### PR TITLE
Fixed alternating row background in dark mode

### PR DESCRIPTION
**Pull request by grolu**
**What this PR does / why we need it**:
Fixed alternating row background in dark mode
Cleaned up dark mode specific styling by using css selector

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```bugfix user
Fixed alternating row background styling in dark mode for worker pools, hibernation schedules and access restrictions
```

### CODE REVIEW

File: frontend/src/components/MainToolbar.vue [hunk 1]
```diff
@@ -267,4 +266,12 @@ export default {
     color: white;
     font-weight: bold;
   }
+
+  .theme--light .tabbar-background {
+    background-color: white;
+  }
+
+  .theme--dark .tabbar-background {
+    background-color: black;
```
Comment (holgerkoser) [inline frontend/src/components/MainToolbar.vue:275]:
```suggestion
  .theme--light .tabs-bar-background {
    background-color: white;
  }

  .theme--dark .tabs-bar-background {
    background-color: black;
```

File: frontend/src/components/MainToolbar.vue [hunk 2]
```diff
@@ -138,7 +138,7 @@ SPDX-License-Identifier: Apache-2.0
       </v-menu>
     </div>
     <template v-if="tabs && tabs.length > 1" v-slot:extension>
-      <v-tabs slider-color="primary darken-3" :background-color="darkTheme ? 'black' : 'white'">
+      <v-tabs slider-color="primary darken-3" class="tabbar-background">
```
Comment (holgerkoser) [inline frontend/src/components/MainToolbar.vue:141]:
```suggestion
      <v-tabs slider-color="primary darken-3" class="tabs-bar-background">
```

File: frontend/src/components/VendorIcon.vue [hunk 3]
```diff
@@ -25,14 +24,11 @@ export default {
       type: Number,
       default: 24
     },
-    lightBackground: {
+    noBackground: {
```
[the remark this answers is not in the document]
Comment (grolu) [reply frontend/src/components/VendorIcon.vue:27]: default is false but I can add it if you want

[the remark this answers is not in the document]
Comment (holgerkoser) [reply frontend/src/components/VendorIcon.vue:27]: **Explicit is better than implicit**
https://en.wikipedia.org/wiki/Zen_of_Python#:~:text=Explicit%20is%20better%20than%20implicit.

File: frontend/src/components/MainToolbar.vue (hunk 2 above)
Comment (holgerkoser) [reply frontend/src/components/MainToolbar.vue]: You resolved the conversation without any comments.

**Review by holgerkoser — APPROVED** [review]
/lgtm

**Review by petersutter — APPROVED** [review]
/lgtm

**Pull request merged**